wallpaper_rs/src/linux.rs

```diff
@@ -55,7 +55,11 @@ impl Desktop for DesktopEnvt {
             }
 
             DesktopEnvt::MATE => {
-                let mate_path = &path[7..];
+                let path_unquoted = enquote::unquote(&path).unwrap();
+                let mate_path = path_unquoted
+                    .strip_prefix("file://")
```

Comment:
Don't you need the prefix for this?

Reply:
I don't actually know, I just replaced all the lines containing `&path[7..];` with the fix

Reply:
Nope, for GNOME, Cinammon and MATE, you need the quotation marks. Just don't change anything other than that for the XFCE environment.

Reply:
Sure, I will revert the change. Still, slicing the beginning of the string like that will remove the quotation mark at the beginning so I think it should be something like
```rust
let mate_path = path.replace("file://", "");
```

Reply:
Nope, you need the ```file://``` prefix.

Reply:
The why was it sliced in the first place? Should I remove the thing altogether or just revert it?

Reply:
Oh wait! I got confused. Yes, the new fix you pushed with ```path.replace``` should be ok as we still preserve the quotes. I thought we were talking about GNOME

Reply:
Cool! Pushing the commit rn